tox.ini

```diff
@@ -3,3 +3,6 @@ envlist = py26, py27, py32, py33, py34
 
 [testenv]
 commands = python tests.py
+
+[flake8]
+exclude=docs/conf.py
```

Comment:
I'm happy to put this wherever. I had to change it slightly for the doctests --

objgraph has a lot of doctests that are partial. For example:

>>> at(id(obj)) is obj

flake will complain that obj is undefined. So I set it up so that there are two flake environments, one with doctests and one without. The run with doctests can add additional ignores and allow the linter for code to remain more strict.